components/Portfolio/AssetDisplay/AssetDisplay.tsx

```diff
@@ -1,36 +1,221 @@
-import type { Asset } from "@/utils/types";
+import type {
+  Asset,
+  AssetCurrent,
+  AssetHistory,
+} from "@/utils/types";
 
```

Comment:
Double-check that the imports are imported according to the criteria set by Cristan and best practices, including absolute imports, components, utilities, and styling.